templates/components/form/fields_macros.html.twig

```diff
@@ -969,3 +969,14 @@
       {{ helper|raw }}
    </label>
 {% endmacro %}
+
+{% macro buttonField(btlabel, btname, bttype, btclass = '', bticon = '', options = {}) %}
+   {% set options = options %}
+   <button class='{{btclass}}' type='{{bttype}}' name='{{btname}}'>
```

Comment:
Please always try to use `"` to surround HTML attributes. It does not change much, but since we started migration to Twig, we try to use `"` for this.

```suggestion
   <button class="{{ btclass }}" type="{{ bttype }}" name="{{ btname }}">
```